packages/analytics/analytics-utilities/src/types/explore/basic.ts

```diff
@@ -22,8 +22,9 @@ export type FilterableBasicExploreDimensions = typeof filterableBasicExploreDime
 
 export interface BasicExploreFilter {
   type: ExploreFilterTypesV2
-  dimension: FilterableBasicExploreDimensions
-  values: (string | number | null)[]
+  dimension?: FilterableBasicExploreDimensions
+  field?: FilterableBasicExploreDimensions
+  values?: (string | number | null)[]
```

Comment:
Maybe making this optional is weird, but it's optional in advanced for metric filters when we expect `value` instead

Reply:
would it makes sense to make a separate Filter type with the exact members and then do the union type as a list for `filters`? cc @adorack 

like. 
```
export interface MetricFilter {
   metric:
   value:
   type: <metricFilterType>
}

AnyFilter = MetricFilter | BasicExploreFilter
```

then in Explore interface have...? :
```
  filters: AnyFilter[],
```